lua/wikis/esports/Widget/EwcTeamsOverview.lua

```diff
@@ -0,0 +1,173 @@
+---
+-- @Liquipedia
+-- wiki=esports
+-- page=Module:Widget/EwcTeamsOverview
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Json = require('Module:Json')
+local Template = require('Module:Template')
+
+local DataTable = Lua.import('Module:Widget/Basic/DataTable')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local WidgetUtil = Lua.import('Module:Widget/Util')
+local Widget = Lua.import('Module:Widget')
+local Link = Lua.import('Module:Widget/Basic/Link')
+local Icon = Lua.import('Module:Widget/Image/Icon/Fontawesome')
+
+---@class EwcTeamsOverview: Widget
+---@operator call(table): EwcTeamsOverview
+local EwcTeamsOverview = Class.new(Widget)
+
+local GAMES = {
+	EWC2025 = {
+		{lis = 'hok', wiki = 'honorofkings'},
+		{lis = 'codbo6', wiki = 'callofduty'},
+		{lis = 'codwz', wiki = 'callofduty'},
+		{lis = 'ff', wiki = 'freefire'},
+		{lis = 'sf6', wiki = 'fighters'},
+		{lis = 'dota2', wiki = 'dota2'},
+		{lis = 'r6s', wiki = 'rainbowsix'},
+		{lis = 'cs2', wiki = 'counterstrike'},
+		{lis = 'pubgm', wiki = 'pubgmobile'},
+		{lis = 'apex', wiki = 'apexlegends'},
+		{lis = 'fc25', wiki = 'easportsfc'},
+		{lis = 'ow2', wiki = 'overwatch'},
+		{lis = 'mlbb', wiki = 'mobilelegends'},
+		{lis = 'mwi', wiki = 'mobilelegends'},
+		{lis = 'rl', wiki = 'rocketleague'},
+		{lis = 'chess', wiki = 'chess'},
+		{lis = 'rennsport', wiki = 'simracing'},
+		{lis = 'pubg', wiki = 'pubg'},
+		{lis = 'cf', wiki = 'crossfire'},
+		{lis = 'ffcotw', wiki = 'fighters'},
+		{lis = 'lol', wiki = 'leagueoflegends'},
+		{lis = 'tft', wiki = 'tft'},
+		{lis = 'valorant', wiki = 'valorant'},
+		{lis = 't8', wiki = 'fighters'},
+		{lis = 'sc2', wiki = 'starcraft2'},
```

Comment:
unrelated to the PR, but sad that ewc is the only big event we get this year (and ewc was abysmal last year)